yml/cdb_latiss.yaml

```diff
@@ -0,0 +1,829 @@
+    length: 64
+    description: Start time of the exposure at the fiducial center of the focal plane
+      array, TAI, accurate to 10ms.
+  - name: obsStartMJD
+    "@id": "#Exposure.obsStartMJD"
+    datatype: double
+    description: Start of the exposure in MJD, TAI, accurate to 10ms.
+  - name: obsEnd
+    "@id": "#Exposure.obsEnd"
+    datatype: timestamp
+    length: 64
+    description: End time of the exposure at the fiducial center of the focal plane
+      array, TAI, accurate to 10ms.
+  - name: obsEndMJD
+    "@id": "#Exposure.obsEndMJD"
+    datatype: double
+    description: End of the exposure in MJD, TAI, accurate to 10ms.
+  - name: expTime
+    "@id": "#Exposure.expTime"
+    datatype: double
+    description: Spatially-averaged duration of exposure, accurate to 10ms.
+  - name: shutTime
+    "@id": "#Exposure.shutTime"
+    datatype: double
+    description: Spatially-averaged shutter-open dduration, accurate to 10ms.
+  - name: darkTime
+    "@id": "#Exposure.darkTime"
+    datatype: double
+    description: Duration from last clear to readout, accurate to 10ms.
+  - name: group_id
+    "@id": "#Exposure.group_id"
+    datatype: string
+    length: 64
+    description: Identifier for the group that this exposure is part of.
+  - name: curindex
+    "@id": "#Exposure.curindex"
+    datatype: int
+    description: Number (1-based) of the observation within its group.
+  - name: maxindex
+    "@id": "#Exposure.maxindex"
+    datatype: int
+    description: Expected number of observations within the group.
+  - name: imgtype
+    "@id": "#Exposure.imgtype"
+    datatype: string
+    length: 64
+    description: Type of exposure taken.
+  - name: emulated
+    "@id": "#Exposure.emulated"
+    datatype: boolean
+    description: True if the exposure was taken in emulation mode.
+  - name: science_program
+    "@id": "#Exposure.science_program"
+    datatype: string
+    length: 1024
+    description: Science program.
+  - name: observation_reason
+    "@id": "#Exposure.observation_reason"
+    datatype: string
+    length: 1024
+    description: Reason for the observation.
+  - name: target_name
+    "@id": "#Exposure.target_name"
+    datatype: string
+    length: 1024
+    description: Target of the observation.
+  - name: shutterOpenBegin
+    "@id": "#Exposure.shutterOpenBegin"
+    datatype: timestamp
+    length: 64
+    description: Start of shutter opening.  TAI, accurate to 10ms.
+  - name: shutterOpenEnd
+    "@id": "#Exposure.shutterOpenEnd"
+    datatype: timestamp
+    length: 64
+    description: End of shutter opening.  TAI, accurate to 10ms.
+  - name: shutterCloseBegin
+    "@id": "#Exposure.shutterCloseBegin"
+    datatype: timestamp
+    length: 64
+    description: Start of shutter closing.  TAI, accurate to 10ms.
+  - name: shutterCloseEnd
+    "@id": "#Exposure.shutterCloseEnd"
+    datatype: timestamp
+    length: 64
+    description: End of shutter opening.  TAI, accurate to 10ms.
+  - name: airTemp
```

Comment:
Units required

Reply:
My reading of the VOUnits standard is that this would require a conversion to K, as degC is not allowed.  Is that correct?  Does that make this value less usable, if more correct?

Reply:
Left as `degC` in the description with no official unit.